src/Controller/SearchController.php

```diff
@@ -23,13 +23,16 @@ public function index(CaseEntityRepository $caseRepository, PaginatorInterface $
         $qb = $caseRepository->createQueryBuilder('c');
 
         if (null !== $search) {
-            $fieldMatches = $searchService->getFieldMatches($search);
-            if (count($fieldMatches) > 0) {
-                $qb = $searchService->applyFieldSearch($qb, $fieldMatches);
-            }
+            // TODO: Reintroduce if we add searchable fields
+//            $fieldMatches = $searchService->getFieldMatches($search);
+//            if (count($fieldMatches) > 0) {
+//                $qb = $searchService->applyFieldSearch($qb, $fieldMatches);
+//            }
 
             $escapedSearch = $searchService->escapeStringForLike($search, '\\');
 
+            $qb->orWhere('c.caseNumber LIKE :search');
+//            $qb->setParameter(':search', '%'.$escapedSearch.'%');
```

Comment:
Remove this line.

Reply:
Removed in dfb0c208a6b3b0813d72f6486324a689b337c5b3.